qt/applications/workbench/workbench/widgets/settings/test_utilities/settings_model_test_base.py

```diff
@@ -0,0 +1,26 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench
```

Comment:
```suggestion
```

Reply:
I think this is in most of the files, do you want it removed from everywhere (in the settings widget)?

Reply:
Yes please, I think it adds no value